docs/prepare-your-dev-environment.md

```diff
@@ -3,10 +3,16 @@
 This document goes through the development dependencies one requires in order to build the RP code.
 
 ## Software Required
+
 1. Install [Go 1.17](https://golang.org/dl) or later, if you haven't already.
+   1. After downloading follow the [Install instructions](https://go.dev/doc/install), replacing the tar archive with your download.
+   1. Append `export PATH="${PATH}:/usr/local/go/bin"` to your shell's profile file.
 
 1. Configure `GOPATH` as an OS environment variable in your shell (a requirement of some dependencies for `make generate`). If you want to keep the default path, you can add something like `GOPATH=$(go env GOPATH)` to your shell's profile/RC file.
 
+1. Append `export GO111MODULE=auto` to your shell's profile file.
+    1. Read [New module changes in Go 1.16](https://go.dev/blog/go116-module-changes) for more information.
+
```

Comment:
Hmm, I have `GO111MODULE=""` in my environment with everything working. Is this something that is going to be machine-specific?

Reply:
Probably not. I ran `update-go-module-dependencies.sh` after adding `GO111MODULE=auto` in my environment before success, so I wasn't sure which one was the fix (most likely the script). I added both to the doc.